core/operation/src/main/java/uk/gov/gchq/gaffer/operation/impl/GetWalks.java

```diff
@@ -54,6 +54,7 @@
     private Iterable<? extends EntityId> input;
     private Map<String, String> options;
     private Integer resultsLimit = 1000000;
+    private boolean prune = false;
```

Comment:
Please can you make sure there is an issue in gaffer-tools to add this prune field to the python shell.